### ISSUE TITLE
Android tablets should display Place Page on the side, not like on phones

### ISSUE THREAD

**Issue by biodranik** [organicmaps/organicmaps#2483]
OM | MapsMe 
-|-
![image](https://user-images.githubusercontent.com/170263/166063428-b40cf769-810f-42a5-8dd5-b8a0f985a677.png) | ![image](https://user-images.githubusercontent.com/170263/166063454-2be1d98f-cc73-434d-af30-f7fc76d7085e.png)

**Comment by Jean-BaptisteC** [comment]
Maybe doing same thing for Search like in Google Maps or Apple Plans ?

**Comment by biodranik** [comment]
Any screenshots to compare?

**Comment by Jean-BaptisteC** [comment]
| OM | Google Maps |
|---|---|
| <img src="https://user-images.githubusercontent.com/87148630/201329356-87873322-a50a-4fc8-a211-b07de081f47b.jpg" height=250/> | <img src="https://user-images.githubusercontent.com/87148630/201329498-83e34f0d-4c0e-44d6-97ef-3318bceccb5a.jpg" height="250"/>

Screenshots are take in Samsung Galaxy Tab S2 Lite.
I don't have Ipad to take screenshots on Apple Plans.

**Comment by biodranik** [comment]
Thanks! What about landscape mode?

**Comment by Jean-BaptisteC** [comment]
Like in smartphone in portrait mode

**Comment by biodranik** [comment]
Buttons are also visible in the landscape mode:
![Screenshot_2023-05-07-22-00-55](https://user-images.githubusercontent.com/170263/236700750-96cb52c8-8854-4523-a06e-60f1d7e0903c.png)

@arnaudvergnet is it hard to fix PP in the landscape?

**Comment by arnaudvergnet** [comment]
I'll look into it when I get the time. If we need to show the search on top of the map, we'll need to do some more refactoring :). The search is in its own activity right now, so we'll have to move it in a fragment instead. But this will be a bit tricky to do if there are any external links that open the search directly as we need to keep that feature.

**Comment by biodranik** [comment]
Yeah, let's plan it first carefully. Aren't we opening all external intents/links from SplashActivity now?

**Comment by arnaudvergnet** [comment]
Seems like it but the logic is to start different activities depending on the start intent/link. So we'll need to rework this logic to open the main map activity with the search fragment already open.

**Comment by arnaudvergnet** [comment]
If we validate the compose migration, we could wait for it to progress so we make this design change during the refactoring.
